src/lexers/javascript-lexer.js

```diff
@@ -1,39 +1,57 @@
-import * as acorn from 'acorn-jsx'
-import injectAcornObjectRestSpread from "acorn-object-rest-spread/inject"
+import * as acorn from 'acorn'
+import injectAcornStage3 from "acorn-stage3/inject"
 import injectAcornEs7 from "acorn-es7"
 import * as walk from 'acorn/dist/walk'
 import BaseLexer from './base-lexer'
 
+const WalkerBase = Object.assign({}, walk.base, {
+  Import(node, st, c) {
+    // We need this catch, but we don't need the catch to do anything.
+  }
+})
```

Comment:
Import is for dynamic imports. Following the pattern in jsx-lexer of putting extensions in a const before the lexer.